test-fixtures/generate-metrics.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+# set -x # For debug
+set -e
+
+BUILD_LOG_FILE="$1"
+TYPE_LOG_FILE="$2"
+THREESHOLD_UNIT_TEST=310
+THREESHOLD_XCUITEST=850
+
+
+WARNING_COUNT=`egrep '^(\/.+:[0-9+:[0-9]+:.|warning:|ld: warning:|<unknown>:0: warning:|fatal|===)' "$BUILD_LOG_FILE" | uniq | wc -l`
```

Comment:
These are the warnings count: https://github.com/mozilla-mobile/firefox-ios/pull/9776#issuecomment-1013248450